kf_lib_data_ingest/validation/values.py

```diff
@@ -0,0 +1,236 @@
+from dateutil.parser import parse as parsedate
+
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+
+
+MIN_AGE_DAYS = 0
+# Max value chosen due to HIPAA de-identification standard
+# Using safe harbor guidelines
+# Equates to 90 years
+MAX_AGE_DAYS = 32872
+NULL_VALUES = {
+    constants.COMMON.NOT_REPORTED,
+    constants.COMMON.CANNOT_COLLECT,
+    constants.COMMON.NOT_APPLICABLE,
+    constants.COMMON.UNKNOWN,
+    constants.COMMON.NOT_AVAILABLE,
+}
+
```

Comment:
Can you add a convenience function here which does validation for a given key and value. Something like:

```
def validate(key, value):
    try:
        desc, funct = INPUT_VALIDATION[key]
    except KeyError:
        raise KeyError(
            f'No validation function found for `{key}` in {__file__}.'
        )

    return funct(value)
```

Reply:
I'm opposed to this. Determination of which validations to run will be based on the contents of the INPUT_VALIDATION table anyway, so there will never be a key error.

Reply:
I'm not sure what to do here. Could use `INPUT_VALIDATION.get(key)` as a compromise?

Reply:
Oh I see so you're going to iterate over keys in `INPUT_VALIDATION` and run every function on the data?

Reply:
yes